.travis.yml

```diff
@@ -56,7 +56,6 @@ notifications:
           - http://julia.mit.edu:8000/travis-hook
 before_install:
     - make check-whitespace
-    - JULIA_SYSIMG_BUILD_FLAGS="--output-ji ../usr/lib/julia/sys.ji"
```

Comment:
> A replacement for this test, rather than deletion

I've gradually (and with no fanfare) been building out support for breaking the build down into component pieces rather than doing it as a monolithic step as this test expected. There's actually not too many pieces left to fall, which I why I keep trying to topple this one.